fenix/app/src/test/java/org/mozilla/fenix/library/bookmarks/DesktopFoldersTest.kt

```diff
@@ -42,6 +43,7 @@ class DesktopFoldersTest {
     fun setup() {
         context = spyk(testContext)
         every { context.components.core.bookmarksStorage } returns mockk()
+        coEvery { context.components.core.bookmarksStorage.countBookmarksInTrees(any()) } returns 0U
```

Comment:
This line will throw errors if someone tries to make an assertion that executes this line in a future test. It might be best to remove it to avoid a footgun in the future.